components/scream/src/physics/p3/atmosphere_microphysics.hpp

```diff
@@ -174,10 +176,10 @@ class P3Microphysics : public AtmosphereProcess
         // Update the atmospheric temperature and the previous temperature.
         const Spack opmid(pmid(icol,ipack));
         const Smask opmid_mask(!isnan(opmid) and opmid>0.0);
```

Comment:
Same considerations about checks and `oblah` names as in the preamble hold for the postprocessing too.